test/run_envoy_fuzz_coverage.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${SRCDIR}" ]] && SRCDIR="${PWD}"
+
+echo "Starting run_envoy_fuzz_coverage.sh..."
+echo "    PWD=$(pwd)"
+echo "    SRCDIR=${SRCDIR}"
+
+# This is the fuzz target that will be run to generate coverage data.
+if [[ $# -gt 0 ]]; then
+  FUZZ_TARGETS=$*
+else
+  # By default, this will be all fuzz targets.
```

Comment:
Something like this also exists in the OSS-fuzz runner script. Could we define this in a common bash utility script and import it in both locations?

Reply:
What do you think of the tag? makes it an easy bazel query